index.html

```diff
@@ -1,14 +1,52 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Engage Bakery</title>
-    <link rel="icon" href="https://www.engage.veented.com/wp-content/uploads/2017/11/cropped-favicon-32x32.png" sizes="32x32">
+    <link rel="icon" href="https://www.engage.veented.com/wp-content/uploads/2017/11/cropped-favicon-32x32.png"
+        sizes="32x32">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css">
     <link rel="stylesheet" href="css/style.css">
 </head>
+
 <body>
-    
+    <!-- ========== Start navbar ========== -->
```

Comment:
you might need to have a div as main container for all website content, then divide that into 
header => nav
main => sections and more content
footer => footer